main.py

```diff
@@ -21,11 +21,14 @@
 import psutil
 import pkg_resources
 from packaging import version
+import paths
+
+own_dir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Did you forget to remove this? As far as I can see this is an obsolete line, considering own_dir is nowhere used in this file.